platform/admin/ui/src/main/webapp/templates/installer/anonClaims.handlebars

```diff
@@ -0,0 +1,33 @@
+{{!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ --}}
+<h3>Configure anonymous claims attributes</h3>
+<div class="inner-scroll scroll-area white ps-container">
+    <div class="well well-sm white">
+        <div id="warning-container" class="warning-div">
+
+        </div>
+        <div id="anom-claim-input-container">
+            <div id="claims-profiles"></div>
+
+            <!--</form>-->
+            <form class="form-horizontal add-federated-source">
```

Comment:
Removed the unused 'add-federated-source' class. Waiting for everyone to finish reviewing before I push my changes up.